packages/contracts-bedrock/src/L1/SystemConfig.sol

```diff
@@ -194,7 +190,6 @@ contract SystemConfig is OwnableUpgradeable, ReinitializableBase, ISemver {
         Storage.setAddress(L1_CROSS_DOMAIN_MESSENGER_SLOT, _addresses.l1CrossDomainMessenger);
         Storage.setAddress(L1_ERC_721_BRIDGE_SLOT, _addresses.l1ERC721Bridge);
         Storage.setAddress(L1_STANDARD_BRIDGE_SLOT, _addresses.l1StandardBridge);
-        Storage.setAddress(DISPUTE_GAME_FACTORY_SLOT, _addresses.disputeGameFactory);
```

Comment:
We should clear the old `DISPUTE_GAME_FACTORY_SLOT` slot in the `upgrade` method, otherwise integrations that read from that slot will return a wrong, dangling value

Reply:
Sounds good